dannce/engine/generator.py

```diff
@@ -1687,6 +2205,16 @@ def visualize(self, original, augmented):
         input("Press Enter to continue...")
 
     def random_continuous_rotation(self, X, y_3d, max_delta=5):
+        """Summary
```

Comment:
`Rotation augmentation for 3D volumes. Rotates X and y_3d a random amount around z-axis.`

Reply:
```suggestion
        """Rotates X and y_3d a random amount around z-axis.
```